modules/Bio/EnsEMBL/Compara/RunnableDB/PairAligner/FilterDuplicates.pm

```diff
@@ -92,7 +91,7 @@ sub fetch_input {
 sub run
 {
   my $self = shift;
-  $self->filter_duplicates;
+  $self->call_within_transaction( sub { $self->filter_duplicates } );
```

Comment:
Not a Perl expert, but I think you could also do:
```suggestion
  $self->call_within_transaction( \&{ $self->filter_duplicates } );
```

Reply:
I have done it this way to match the rest of the runnables in the repository and it works, so I will leave it this time, thanks for the alternative though, to play with another time :+1: